description/description.rdfs.ttl

```diff
@@ -31,7 +31,7 @@ haDes:NewspaperIssuePage   a   rdfs:Class ;
     rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> .
 
 haDes:DVD   a   rdfs:Class ;
-    rdfs:label "Dvd"@nl ;
+    rdfs:label "DVD"@nl ;
```

Comment:
In het NL is het normaliter met kleine letters: https://woordenlijst.org/zoeken/?q=dvd